image/templates/helm/shared/templates/02-scanner-v4-07-indexer-deployment.yaml

```diff
@@ -26,9 +26,6 @@ spec:
         {{- include "srox.podLabels" (list . "deployment" "scanner-v4-indexer") | nindent 8 }}
       annotations:
         {{- $annotations := dict "traffic.sidecar.istio.io/excludeInboundPorts" "8443" -}}
-        {{- if eq ._rox.env.openshift 4 }}
-          {{- $_ := set $annotations "openshift.io/required-scc" "nonroot-v2" -}}
```

Comment:
AFAIK, this annotation is in fact explicitly required for the OpenShift SCC pinning feature to work.
Please reach out to Anjali to check in with her (PM for the pinning feature) to be safe :smile: 

Let's keep it, but instead explicitly require restricted: 
`{{- $_ := set $annotations "openshift.io/required-scc" "restricted-v2" -}}`

We need to update the serviceaccount for scanner to disallow it the usage of `nonroot-v2` to ensure that it cannot be used anymore.
This can be done by removing the `Role` and `RoleBinding` in here:
https://github.com/stackrox/stackrox/blob/mc/scanner-v4-restrict-scc/image/templates/helm/shared/templates/02-scanner-v4-01-security.yaml#L11-L48

Reply:
Thank you.
I have re-added the SCC pinning.

Reply:
I have furthermore added the `restricted-v2` SCC to the role associated with the scanner v4 service account. Until now it was only listing the `nonroot-v2` SCC.

As discussed offline already, the `nonroot-v2` role also needs to stay there for the time being, because in its current form the scanner-v4-db workload requires `nonroot-v2`.

Reply:
Great, thank you for improving!